tests/other/metadce/hello_libcxx_fastcomp_O2_EMULATED_FUNCTION_POINTERS.funcs

```diff
@@ -411,7 +411,6 @@ $___get_locale
 $___intscan
 $___isxdigit_l
 $___loc_is_allocated
-$___munmap
```

Comment:
I couldn't any any good reason why this fastcomp-only change occured.   I looked that bitcode both before and after and I have no clues.   Since it technically and optimization and since its fastcomp-only I'm tempted to let it slide without deeper inspection.